### DISCUSSION TITLE
The version of process-compose I ended up with in my flake didn't know about `--no-server`

### DISCUSSION THREAD

**Discussion by commandodev** [Platonic-Systems/process-compose-flake#71] [Q&A]
I ended up pinning an older revision (which I'd seen working elsewhere):

`github:Platonic-Systems/process-compose-flake/44d260ddba5a51570dee54d5cd4d8984edaf98c2`

Is there a way to specify the version of process-compose which is used?

**Comment by srid** [comment]
I suppose we can add an assert check for this,

https://github.com/Platonic-Systems/process-compose-flake/issues/72
